src/ui/qgsoptionsbase.ui

```diff
@@ -332,7 +332,7 @@
        <item>
         <widget class="QStackedWidget" name="mOptionsStackedWidget">
          <property name="currentIndex">
-          <number>8</number>
+          <number>6</number>
```

Comment:
In the end there are no changes to the options dialog right? It would be nice to revert the changes in the .ui file on this PR...

Reply:
Done.